src/jquery.mask.js

```diff
@@ -467,6 +472,10 @@
         return this.data('mask').getCleanVal();
     };
 
+    $.fn.maskedVal = function(val) {
```

Comment:
Forget what I said about the suggestion of changing the return... what do you think of the name `masked` instead of maskedVal? Im having mixed-feelings about the proposed name.